bob-backend/src/pearl/group.rs

```diff
@@ -613,6 +613,36 @@ impl Group {
         result
     }
 
+    pub(crate) async fn corrupted_blobs_count(&self) -> usize {
+        let mut corrupted_blobs = 0;
+        let holders_guard = self.holders();
```

Comment:
`self.holders` can be used directly here. In this way there is no need in `holders_guard` variable